Dockerfile

```diff
@@ -0,0 +1,9 @@
+ARG CLI_VERSION=latest
+FROM microsoft/powershell:latest
+ARG CLI_VERSION=latest
```

Comment:
Is there a reason why we need to specify the arg twice?

Reply:
See the Gotcha section of the below article

https://www.google.nl/amp/s/schneide.blog/2019/10/21/using-parameterized-docker-builds/amp/

Reply:
Got it. Since we don't need the ARG on line 1 for line 2, wouldn't it be sufficient to skip the one in line 1 and just use the latter one?

Reply:
You would think, but I found that didn't work, it was late when I was doing this so I'll take another look 😴

Reply:
Yes, we don't need line 1, tested tonight and it's not needed.

Reply:
Line removed in last update